apps/freelance/src/redux/services/testing/jobs.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import fetchMock from 'jest-fetch-mock';
+import { Provider } from 'react-redux';
+import { renderHook, waitFor } from '@testing-library/react';
+import { setUser } from 'redux/auth/auth-slice';
+import { authReducer } from 'redux/auth/auth-slice';
+import { jobsApi, useFindUserJobsQuery } from 'redux/services/jobsApi';
+import { setupApiStore } from 'redux/storeTest';
+
+import { testToken, userJobs } from './mock-data';
+
+fetchMock.enableMocks();
+
+describe('check userjob query', () => {
+  const OLD_ENV = process.env;
```

Comment:
why name is OLD_ENV, when I read I expect to see somewhere NEW_ENV but there no new

Reply:
changed